### DISCUSSION TITLE
Gaps in bridge lines

### DISCUSSION THREAD

**Discussion by vicpon** [OrcaSlicer/OrcaSlicer#842] [Q&A]
Slicing 2 different files in Orca Slicer. One has gaps between bridge lines, the other doesn't. What setting is affecting this? Both of these are were sliced with the same settings, just opened Orca, added model and clicked Slice without changing anything. I tried slicing the model in Cura and it doesn't have any gaps.
![FB_IMG_1682302242499](https://user-images.githubusercontent.com/2129528/233987626-2e36c6e8-5681-40b6-a207-ed41e1ea175b.jpg)
![FB_IMG_1682302246171](https://user-images.githubusercontent.com/2129528/233987627-cee7ae62-f4f0-4dfe-8de9-9b5afbb6c62d.jpg)

**Comment by SoftFever** [comment]
Did you check the bridge flow value? That value will affect this visualization.

**Comment by LabunskyA** [comment]
For anyone still wondering: it's due to 'Thick internal bridges'. Orca visualizes them with a default line width, while in actuality the line will be printed somewhere twice as thick

**Reply by igiannakas** [reply]
Thick (and thin) bridges are assumed to be 0.4mm wide as there is no squish beneath them. They can never be wider than the nozzle width.

**Reply by LabunskyA** [reply]
AFAIK, Orca assumes them thicker due to squish in infill / support interface contact points. Mid air they are as wide as the nozzle, but are taller. Why do you think they're called 'thick', after all?
